README.md

```diff
@@ -384,6 +384,32 @@ The schemas mirror the CDF API as closely as possible.
 | `metadata`           | `map(string, string)` | Yes       |
 | `dataSetId`          | `long`                | Yes       |
 
+### Sequences
+| Column name          | Type                  |  Nullable |
+| ---------------------| ----------------------| --------- |
+| `externalId`         | `string`              | Yes        |
+| `name`   | `string`              | Yes        |
+| `description`             | `string`              | Yes       |
+| `assetId`               | `long`              | Yes        |
+| `metadata`           | `map(string, string)` | Yes       |
+| `dataSetId`          | `long`                | Yes       |
+| `columns`        | `array(SequenceColumn)`              | No       |
+
+The `columns` field should be an array of `SequenceColumn`s, which are rows with the following fields:
```

Comment:
I don't know how to create that after reading this... what would that look like in SQL? Could we provide an example?

I kind of wonder if it wouldn't be a lot more user friendly to just use an array of maps to describe the columns?

Reply:
I have added an example, not sure if I can link it from here.